static/app/views/insights/browser/webVitals/components/pageOverviewWebVitalsDetailPanel.tsx

```diff
@@ -479,7 +477,7 @@ export function PageOverviewWebVitalsDetailPanel({
             />
           ) : (
             <GridEditable
-              data={transactionsTableData}
+              data={transactionsTableData as any as TransactionSampleRowWithScore[]} // TODO: fix typing
```

Comment:
👀 it looks like you've removed `projectSlug` from the responses (which I agree with!) is that what's causing the type issues? If yes, would it make sense to remove `projectSlug` in a separate PR and clean up all the types at the same time?

Reply:
Yeah i'd rather keep that in another PR